db/sql/cassandra/plugin_impl_cassa.go

```diff
@@ -15,11 +15,20 @@
 package cassandra
 
 import (
+	"github.com/gocql/gocql"
+	"github.com/ligato/cn-infra/core"
 	"github.com/ligato/cn-infra/db/sql"
 	"github.com/ligato/cn-infra/flavors/local"
+	"github.com/ligato/cn-infra/health/statuscheck"
 	"github.com/willfaught/gockle"
 )
 
+// HealthCheck is a structure used to represent table to help verify health status
+type HealthCheck struct {
```

Comment:
Just do some read-only query using broker.Exec(). It is too much to have this not needed table on a particular schema.